### ISSUE TITLE
Suggestions for Additions/Changes to Races

### ISSUE THREAD

**Issue by rexthepig** [queryluke/masseffect-5e#400]
Good job with v0.9.2, I know you were unable to add everything you wanted, but the things you did add are great. 
I thought I might share a few ideas for additions/changes to the races:

The hanar having a hovering speed seems a bit overpowered, even with the few disadvantages they have. An alternative way to represent the hanar's contra-gravitic levitation packs would be for them to ignore difficult terrain and perhaps constantly behaving as if under the effect of the _Feather Fall_ spell. It could also be possible for the pack to be deactivated/broken, which would reduce the hanar's movement speed to about 2 meters but also make them small, allowing them to enter smaller spaces.

The elcor already have proficiency in insight and perception, but maybe they could also be proficient in deception, I think it would be hard to tell if an elcor was lying. Maybe deception could replace perception, as, whilst perception does make sense, insight and deception both relate to the elcor's monotone speech pattern, elcor are good at reading the emotions of other races and hiding their own.

Should the consequences for when a volus' hermetic suit is breached not be more severe? Whilst quarians just get sick, according to the wiki-
> Traditional nitrogen/oxygen air mixtures are poisonous to [volus], and in the low pressure atmospheres tolerable to most species, their flesh will actually split open.

Perhaps they should take a combination of poison and slashing damage for each turn the suit remains breached.

Whilst on the topic of hermetic suits, I think it would make sense if acid damage was also capable of breaching them.
Also, the quarian's ocular augmentation option is still described as "enchanted orbs".

It seems weird to me that the unshackled AI is a subrace for the geth, as the two have pretty much nothing in common except for the fact that they're both synthetic. If they have to be grouped together, maybe they could both be subraces for an AI race? I don't know, this is more for lore reasons than gameplay, mechanically this works fine.

Again, great job, I can't wait for the new spells and monsters hopefully arriving in the next update.

**Comment by queryluke** [comment]
@rexthepig Thanks! I appreciate the feedback.

## Hanar
I went digging back into the 5e PHB, and realized my mistake. __Hover__ is an extension of flying not it's own type of movement. Which means the current Hanar speed suggests that they can _fly_ and _hover_. This was not my intent.  I wanted hanar to be able to:
1. Avoid terrain challenges because they hover over it (great minds)
2. Avoid being knocked prone because they are hovering in the air (which is part of the PHB rule and where I got confused).

I think the best way to convey this rule is to give them a swim speed and race-specific device. This would allow an expanded explanation of the rules I intended.

## Elcor
Another good point. It might make sense to break up the _Careful_ trait. @bpmcpherson , since you created the original, do you have any thoughts?

In addition, elcor could have _Keen Sense of Smell_ (advantage on perception checks that rely on smell), since they have highly attuned olfactory senses.

## Volus
I saw this as well. But I was being lazy and wanted to reuse the data file. But the suits are different. Primarily, volus suits are pressurized which might have some additional mechanics.

## Acid & Hermetic suits
Logically it makes sense. My concern is that it opens up discussion for other damage types. If the acid is melting through the suit, could fire damage do the same (I know a chemical burn is different)? What about cold damage making it brittle and causing it to crack? What about bludgeoning damage to the helmet/face mask. which could cause a crack. But I also want to avoid a long list of potential damage types and situations. I'd rather it be all damage types (which it was) or just the two common ones.

## Quarian's "Enchanted Orbs"
Whoops! Nice catch. I missed that phrase when I was bringing some of the information over from Races from the Relay. I'll make sure that get's fixed.

## Unshackled AI
I would call them a "Variant" race, rather than a subrace. My reasoning for this change is that the Unshackled AI aren't really a race. Specifically it refers to EDI getting a body, then was referenced in the multiplayer with the Alliance Infiltration Units, and there is a page on [AI](https://masseffect.fandom.com/wiki/Artificial_Intelligence). So, to me, Unshackled AI are unique and don't have an underlying set of racial traits. In addition, the traits they did have were the same as the Geth.

In short, I really liked the idea of someone getting to play an Unshackled AI, but it seemed like overkill to make them a unique race. Thus, you can use the Geth traits (which reflect their "construct-ness", but you're actually a unique AI. I can try to make this a bit more clear in the description, specifically calling out that you are not a hive mind but a single-entity AI.

**Comment by rexthepig** [comment]
@queryluke Yeah, I've seen a few people make that mistake about hovering.

About the hermetic suits, I think it's a good idea to go with simplicity over realism in this case. Maybe you could add a bit about the GM being able to decide if other circumstances would breach the suit, but that seems a little too open ended.

And with the unshackled AI, I think that makes sense, and I'm sure making the description a bit more clear will sort out any confusion.

On a related note, maybe you should include the option of a subrace in the summary that appears on the races page.

**Comment by rexthepig** [comment]
Also, I was wondering if there are still plans for an awakened collector subrace, or was that scrapped?

**Comment by queryluke** [comment]
@rexthepig Oh, man I knew I forgot about a subrace. Yes, Awakened Collector is still on the docket. @bpmcpherson had some ideas in #218. I'll try to get that one in the next release.

**Comment by queryluke** [comment]
included all of these in v1
